install-dev/data/xml/hook.xml

```diff
@@ -57,6 +57,9 @@
     <hook id="displayTop">
       <name>displayTop</name><title>Top of pages</title><description>This hook displays additional elements at the top of your pages</description>
     </hook>
+    <hook id="displayFullWidthTop">
+      <name>displayFullWidthTop</name><title>Top of pages without width limitation.</title><description>This hook displays additional elements at the top of your pages without width limitation.</description>
+    </hook>
```

Comment:
Hello @iqit-commerce 

Missing sql update into 1.7.0.0.sql nop ?

INSERT INTO PREFIX_hook(name,title,description,position) VALUES ('displayFullWidthTop', 'Top of pages without width limitation.', 'This hook displays additional elements at the top of your pages without width limitation.', '1');